File: dotNet/Semantria/Mapping/Output/CollTopic.cs
using System;
using System.Collections.Generic;
using System.Runtime.Serialization;
using System.Xml.Serialization;

namespace Semantria.Com.Mapping.Output
{
    [DataContract(Name = "topic")]
    [XmlRootAttribute("topic", Namespace = "")]
    public class CollTopic
    {
        [DataMember(Name = "title")]
        [XmlElementAttribute("title")]
        public string Title { get; set; }

        [DataMember(Name = "type")]
        [XmlElementAttribute("type")]
        public string Type { get; set; }

        [DataMember(Name = "hitcount")]
        [XmlElementAttribute("hitcount")]
        public int Hitcount { get; set; }

        [DataMember(Name = "sentiment_score")]
        [XmlElementAttribute("sentiment_score")]
        public float SentimentScore { get; set; }

        [DataMember(Name = "label")]
        [XmlElementAttribute("label")]
        public string Label { get; set; }

        [DataMember(Name = "sentiment_polarity")]
        [XmlElementAttribute("sentiment_polarity")]
        public string SentimentPolarity { get; set; }
    }
}

File: Java/src/main/java/com/semantria/mapping/output/CollTopic.java
package com.semantria.mapping.output;

import javax.xml.bind.annotation.XmlElement;

public class CollTopic
{
    private String title;
    private String type;
    private Integer hitcount;
    private Float sentiment_score;
	private String sentiment_polarity;

	@XmlElement(name="title")
    public String getTitle() { return title; }
    @XmlElement(name="type")
    public String getType() { return type; }
    @XmlElement(name="hitcount")
    public Integer getHitCount() { return hitcount; }
    @XmlElement(name="sentiment_score")
    public Float getSentimentScore() { return sentiment_score; }
	@XmlElement(name = "sentiment_polarity")
	public String getSentimentPolarity() { return sentiment_polarity; }

    public void setTitle(String title) { this.title = title; }
    public void setType(String type) { this.type = type; }
    public void setHitCount(Integer hitCount) { this.hitcount = hitCount; }
    public void setSentimentScore(Float sentiment_score) { this.sentiment_score = sentiment_score; }
	public void setSentimentPolarity(String sentiment_polarity){ this.sentiment_polarity = sentiment_polarity; }
}
